File: eco-core/core-proxy/src/main/java/com/willfp/ecoenchants/proxy/proxies/EcoCraftEnchantmentManagerProxy.java
package com.willfp.ecoenchants.proxy.proxies;

public interface EcoCraftEnchantmentManagerProxy {
    /**
     * Re-Register new CraftEnchantments for all vanilla enchantments in order to modify their max level.
     */
    void registerNewCraftEnchantments();
}

File: eco-core/core-proxy/src/main/kotlin/com/willfp/ecoenchants/proxy/proxies/EcoCraftEnchantmentManagerProxy.kt
package com.willfp.ecoenchants.proxy.proxies

import com.willfp.ecoenchants.vanilla.VanillaEnchantmentData
import org.bukkit.NamespacedKey
import org.bukkit.enchantments.Enchantment

interface EcoCraftEnchantmentManagerProxy {
    fun registerNewCraftEnchantment(enchantment: Enchantment, data: VanillaEnchantmentData)
}
